tekton-catalog/any-sequencer/Makefile

```diff
@@ -24,10 +24,10 @@ init:
 	mkdir -p ${BIN_DIR}
 
 local: init
-	go build -o=${BIN_DIR}/any-taskrun ./cmd
+	go build -o=${BIN_DIR}/any-taskrun
 
 build-linux: init
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o=${BIN_DIR}/any-taskrun ./cmd
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o=${BIN_DIR}/any-taskrun
```

Comment:
do we need to update the name `any-taskrun`? since we consider conditions.

Reply:
I'm indifferent but think we should still keep it simple, while open to good names if have.